### DISCUSSION TITLE
Добавление ресурсов в README

### DISCUSSION THREAD

**Discussion by egorprnn** [MrZillaGold/VK2Discord#231] [Общее]
Вы можете запросить добавление ресурса, который использует **VK2Discord** в [**README**](https://github.com/MrZillaGold/VK2Discord/blob/master/README.md) проекта.

## 📌 Требования
Ресурс должен содержать адекватный контент, а также полностью соответствовать одной из группе требований:

### Discord
* Наличие 1 000 участников или партнёрского статуса

### ВКонтакте
* Наличие 1 000 подписчиков или верификации

## 📎 Необходимые материалы

* Заголовок ресурса.
* Аватар в соотношении **1:1** хорошего качества.
* Ссылка на группу или **бесконечное** по времени и количеству использований приглашение на сервер Discord. Пользовательские ссылки не принимаются, так как они могут стать недействительными после истечения бустов сервера.

**Comment by TheDarknessRU** [comment]
Канал Morsy
Аватар:
https://cdn.discordapp.com/attachments/420311284035223572/899071151987376168/Morsy.png
Ссылка на Discord:
https://discord.com/invite/morsy

**Reply by egorprnn** [reply]
Добавил

**Reply by DUB1401** [reply]
Приглашение не действительно.

**Comment by DmitrySlyozka** [comment]
Канал BORDERLANDS
Ссылка на аватар: https://i.ibb.co/qDqHsTd/6.jpg
Ссылка на VK: https://vk.com/borderlands
Ссылка на Discord: https://discord.gg/nKd48js2fG

**Reply by egorprnn** [reply]
Спасибо, добавил

**Comment by Grishustrik** [comment]
Используется для фандом сообщества и дискорд сервера по аниме и манге Одиноки Рокер!

ВК:https://vk.com/bocchi_the_rock
ДС:https://discord.gg/HFxQT2pB6X
